src/containers/pages/IRS/JurisdictionsReport/index.tsx

```diff
@@ -0,0 +1,290 @@
+import DrillDownTable, { hasChildrenFunc } from '@onaio/drill-down-table';
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import superset, { SupersetFormData } from '@onaio/superset-connector';
+import React, { useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { connect } from 'react-redux';
+import { RouteComponentProps } from 'react-router';
+import 'react-table/react-table.css';
+import { Col, Row } from 'reactstrap';
+import { Store } from 'redux';
+import IRSTableCell from '../../../../components/IRSTableCell';
+import HeaderBreadcrumb from '../../../../components/page/HeaderBreadcrumb/HeaderBreadcrumb';
+import Loading from '../../../../components/page/Loading';
+import {
+  SUPERSET_IRS_REPORTING_FOCUS_AREAS_COLUMNS,
+  SUPERSET_IRS_REPORTING_JURISDICTIONS_COLUMNS,
+  SUPERSET_IRS_REPORTING_JURISDICTIONS_DATA_SLICES,
+  SUPERSET_IRS_REPORTING_JURISDICTIONS_FOCUS_AREA_LEVEL,
+  SUPERSET_IRS_REPORTING_PLANS_SLICE,
+} from '../../../../configs/env';
+import { HOME, HOME_URL, IRS_REPORTING_TITLE, REPORT_IRS_PLAN_URL } from '../../../../constants';
+import '../../../../helpers/tables.css';
+import { FlexObject, RouteParams } from '../../../../helpers/utils';
+import supersetFetch from '../../../../services/superset';
+import GenericJurisdictionsReducer, {
+  fetchGenericJurisdictions,
+  GenericJurisdiction,
+  getGenericJurisdictionsArray,
+  reducerName as GenericJurisdictionsReducerName,
+} from '../../../../store/ducks/generic/jurisdictions';
+import IRSPlansReducer, {
+  fetchIRSPlans,
+  getIRSPlanById,
+  IRSPlan,
+  reducerName as IRSPlansReducerName,
+} from '../../../../store/ducks/generic/plans';
+import { getJurisdictionBreadcrumbs } from '../Map/helpers';
+import { IRSTableColumns } from './helpers';
+import './style.css';
+
+/** register the reducers */
+reducerRegistry.register(IRSPlansReducerName, IRSPlansReducer);
+reducerRegistry.register(GenericJurisdictionsReducerName, GenericJurisdictionsReducer);
+
+const slices = SUPERSET_IRS_REPORTING_JURISDICTIONS_DATA_SLICES.split(',');
+
+/** IRS Jurisdictions props */
+export interface GenericJurisdictionProps {
+  fetchJurisdictions: typeof fetchGenericJurisdictions;
+  fetchPlans: typeof fetchIRSPlans;
+  hasChildren: typeof hasChildrenFunc;
+  jurisdictions: GenericJurisdiction[] | null;
+  plan: IRSPlan | null;
+  service: typeof supersetFetch;
+}
+
+/** Renders IRS Jurisdictions reports */
+const JurisdictionReport = (props: GenericJurisdictionProps & RouteComponentProps<RouteParams>) => {
+  const [jurisdictionId, setJurisdictionId] = useState<string | null>(
+    props.match && props.match.params && props.match.params.jurisdictionId
```

Comment:
instead of the ternary, you could wrap the multiple conjunctive clause in parentheses and after closing paren suffix with `|| null`